app/src/main/res/values/strings.xml

```diff
@@ -6,5 +6,6 @@
     <string name="nav_schedule">Schedule</string>
     <string name="nav_map">Map</string>
     <string name="nav_shuttle">Shuttle</string>
+    <string name="default_web_client_id">219414920184-vo13n4ghkndsilo4t3b8dc88oqj5i1n7.apps.googleusercontent.com</string>
```

Comment:
What is this string for? Where is it used? Can we change it because it is unclear what these numbers mean. Can you clarify @ManpreetSohal

Reply:
This is the project's web client id used to sign in.